packages/wrangler/src/pages/buildFunctions.ts

```diff
@@ -91,39 +92,39 @@ export async function buildFunctions({
 	});
 
 	const absoluteFunctionsDirectory = resolve(functionsDirectory);
+	let bundle: BundleResult;
 
 	if (plugin) {
-		RUNNING_BUILDERS.push(
-			await buildPlugin({
-				routesModule,
-				outfile,
-				minify,
-				sourcemap,
-				watch,
-				nodeCompat,
-				functionsDirectory: absoluteFunctionsDirectory,
-				local,
-				betaD1Shims: d1Databases,
-				onEnd,
-			})
-		);
+		bundle = await buildPlugin({
```

Comment:
@GregBrimble because you have more context than me here ... I know we said we leave plugins untouched for now. Do we maybe want to turn this feat on for plugin authors in a separate PR?

Reply:
discussed this IRL, and we concluded we'll park this for now. resolving...